src/content/blog/astro-570.mdx

```diff
@@ -0,0 +1,273 @@
+---
+title: 'Astro 5.7'
+description: 'Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.'
+homepageLink:
+  title: 'Astro 5.7'
+  subtitle: 'Available now!'
+publishDate: '2025-04-15'
+authors:
+  - matt
+  - florian
+  - ema
+  - nate
+coverImage: '/src/content/blog/_images/astro-570/blog-post-5-7.webp'
+socialImage: '/src/content/blog/_images/astro-570/og-astro-5-7.webp'
+lang: 'en'
+---
+
+import BlogContentImage from '/src/components/BlogContentImage.astro';
+import Note from '~/components/Note.astro';
+import Mention from '/src/components/Mention.astro';
+
+**Astro 5.7 has a basketload of treats, including stable Sessions and SVG components and a new Experimental Fonts API.**
+
+🐣 Full of the joys of northern-hemisphere spring, Astro 5.7 is here with goodies galore:
+
+- [**Experimental Fonts API**](#experimental-fonts-api)
+- [**Sessions API**](#sessions-api)
+- [**SVG Components**](#svg-components)
+- [**Config Imports**](#config-imports)
+
+To upgrade an existing project, use the automated `@astrojs/upgrade` CLI tool. Alternatively, upgrade manually by running the upgrade command for your package manager:
+
+```sh
+# Recommended:
+npx @astrojs/upgrade
+
+# Manual:
+npm install astro@latest
+pnpm upgrade astro --latest
+yarn upgrade astro --latest
+```
+
+## Experimental Fonts API
+
+Almost every website these days use custom fonts. However, it can be surprisingly complicated to get them right: how many font faces should I include? Why do I get a Flash of Unstyled Text? Should I preload?
+
+Fortunately, web APIs have greatly improved recently and thanks to amazing initiatives of the Open-Source community (eg. [`unifont`](https://github.com/unjs/unifont/) and [`capsize`](https://github.com/seek-oss/capsize)), it's now easier than ever to use fonts in Astro!
+
+This experimental feature allows you to use fonts from your filesystem and various providers (eg. Google, Fontsource, Bunny...) through a unified API. Keep your site performant thanks to sensible defaults and automatic optimizations.
```

Comment:
```suggestion
This experimental feature allows you to use fonts from both your filesystem and several built-in supported providers (e.g. Google, Fontsource, Bunny) through a unified API. Keep your site performant thanks to sensible defaults and automatic optimizations including fallback font generation.
```
Is this the right way to say it? I think the sentence could use a nod to fallbacks here.